saleor/payment/interface.py

```diff
@@ -316,7 +316,7 @@ class GatewayResponse:
     error: str | None
     customer_id: str | None = None
     payment_method_info: PaymentMethodInfo | None = None
-    # @deprecated
+    # @deprecated instead of storing entire response, pick only needed fields
```

Comment:
You already removed the usage, so you can fully drop the `raw_response` from the interface? Or not?

Reply:
no because other apps are writing there (but other app write non-sensitive fields, so they are not refactored)